src/components/EventCalendarComponent.vue

```diff
@@ -0,0 +1,25 @@
+<script>
+import { EventCalendar } from "@dhx/trial-eventcalendar";
+import "@dhx/trial-eventcalendar/dist/event-calendar.css";
+
+
+export default {
+    props: ["events", "date"],
+
+
+    mounted() {
+        new EventCalendar(this.$refs.cont, {
+            events: this.events,
+            date: this.date,
+        });
+    },
+    unmounted() {
+        this.$refs.cont.innerHTML = "";
```

Comment:
We should call the destructor from here, clearing the container won't release event handlers that may be attached to it or to window https://docs.dhtmlx.com/eventcalendar/api/methods/js_eventcalendar_destructor_method/
(we've missed it in the initial version of the demo)